mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(product) {
+  let tax = 0.2 * product;
+  return tax + product;
+}
```

Comment:
it would be better to replace variable name `product`  with `productPrice` or just `price`.

though next 2 lines are absolutely fine, it might really arguable 
but as per my experience, i would rather see in my project this:
```js
let tax = product * 0.2;
return product + tax;
```
you see,  product goes first.
it seems to be more natural, or maybe more logical.